MANUAL_REGRESSION.md

```diff
@@ -32,7 +32,7 @@
     - user should be able to retake or continue with taken photo
 
 ##### 3. Cross-device with link
-(on Firefox, Safari, IE11 and Microsoft Edge browsers)
+(on private mode of Firefox, Safari, IE11 and Microsoft Edge browsers)
```

Comment:
safari was the problematic one btw. I believe mean private mode for all of the browser, but it might not be explicit.

Reply:
good point, I'll rewrite